pydantic_models/Treatment.py

```diff
@@ -0,0 +1,190 @@
+        examples="100",
+        ge=0)
+    
+    treatmentAnatomicSite: str = Field(
+        description="Anatomical site of administered drug or chemotherapy",
+        examples="arm"
+    )
+
+    treatmentAnatomicalSiteType: Literal["Primary Tumor", "Lymph Node", "Metastatic"] = Field(
+        description="Anatomical site type such as primary, lymph node or metastatic",
+        examples="Metastatic"
+    )
+
+    daysPerCycleOfChemotherapy: int = Field(
+        description="Days per cycle of chemotherapy",
+    )
+
+    numberOfCyclesOfChemotherapy: int = Field(
+        description="The number of cycles of chemotherapy refers to the repeated administration of chemotherapy drugs over a defined period",
+        examples="4",
+        ge=1
+    )
+
+    routeOfAdministration: Literal["Oral", "Intravenous", "Subcutaneous"] = Field(
+        description="The route of administration describes how chemotherapy drugs are delivered into the body",
+        examples="Intravenous"
+    )
+
+    adverseEventList: str = Field(
+        description="Adverse events refer to undesirable or harmful effects experienced by patients as a result of chemotherapy or drug treatments",
+        examples="Nausea"
+    )
+
+    adverseEventTreatment: str = Field(
+        description="Adverse event treatment involves managing and mitigating the side effects and complications associated with drug or chemotherapy",
+        examples="Zofran"
+    )
+
+    locationReactionReported: str = Field(
+        description="Location of Adverse Event",
+        examples="injection site redness"
+    )
+
+    severityReported: Literal["Mild", "Moderate", "Severe", "Life Threatening"] = Field(
+        description="Severity of the adverse event",
+        examples="Mild"
+    )
+
+    previousTreatments: str = Field(
+        description="Previous treatments for the donor"
+    )
+
+    failedTreatments: str = Field(
+        description="Failed treatments of the donor"
+    )
+
+    treatmentSwitch: str = Field(
+        description="Treatment switch information for the donor"
+    )
+
+    drugClass: str = Field(
+        description="The class that drug belongs to in ATC Classification"
+    )
+
+    drugBrandName: str = Field(
+        description="The brand name of the drug in use"
+    )
+
+    drugNameGeneric: str = Field(
+        description="",
+        example="Diazepam"
+    )
+
+    treatmentChronologyWithSurgery: str = Field(
+        description="The order of events for treatment and surgery"
+    )
+
+    treatmentStartDate: date = Field(
+        description="Start date for treatment"
+    )
+
+    treatmentEndtDate: date = Field(
+        description="End date for treatment"
+    )
+
+    isTreatmentOngoing: bool = Field(
+        description="If the patient is currently undergoing treatment or not"
+    )
+
+    isPrimaryTreatment: bool = Field(
+        description="Is the treatment type received primary or not"
+    )
+
+    lineOfTreatment: str = Field(
```

Comment:
if (isPrimaryTreatment == 'yes') : 
Cannot add value to lineOfTreatment because this is primary treatment
else if (isPrimaryTreatment == 'no' && lineOfTreatment <= 1) : 
Value in lineOfTreatment should be greater than 1

Reply:
`line_of_treatment` is assigned a type `str` in the sheet. Can you please clarify the data type to be used?